src/github.com/stellar/gateway/protocols/compliance/auth.go

```diff
@@ -38,8 +38,8 @@ type AuthData struct {
 	NeedInfo bool `json:"need_info"`
 	// The transaction that the sender would like to send in XDR format. This transaction is unsigned.
 	Tx string `json:"tx"`
-	// The full text of the memo the hash of this memo is included in the transaction.
-	Memo string `json:"memo"`
+	// The full text of the attachment. The hash of this attachment is included in the transaction memo.
+	Attach string `json:"attach"`
```

Comment:
It was @jedmccaleb's change: https://github.com/stellar/docs/commit/1ee6bff62e8598196310e521d3f5bb94d01de42a#diff-5eaf03a3c7bbac64863950a2e5c809c0R37